content/en/apps/guides/debugging/secure-sharing-of-developer-instance.md

```diff
@@ -0,0 +1,78 @@
+---
+title: "Securely Sharing Your Development Environment"
+linkTitle: "Securely Sharing Your Development Environment"
+weight: 
+description: >
+  Use a publicly accessible Linux web server to forward https requests to your development environment
+relatedContent: >
+  
+---
+
+{{% alert title="Warning" %}} 
+Be extra careful with this process! The end result will be that your development instance will be accessible to the internet. If you have simple logins and passwords like "admin/test.223" because you thought it was just your local dev instance and it doesn't matter, now it matters! Whenever you're not using the SSH tunnel for testing, shut it down so not remote access is allowed.
+
+Never expose a development instance to the internet where you've replicated production data locally. Well, maybe not never, but with extreme care and intention.
+{{% /alert %}}
+
+## Overview
+When using a local [development environment](https://github.com/medic/cht-core/blob/master/DEVELOPMENT.md), you may want to share your work with other collaborators. You also may need to access the environment from your [mobile device](https://github.com/medic/medic-android/) which requires an SSL certificate (the "s" in "https"). By using a publicly accessible web server, you can receive the secure https requests and forward them back to your CHT instance which doesn't have https set up:
+
+[<img src="/apps/guides/debugging/images/SSH.tunnel.diagram.svg" width=100% height=100%>](/apps/guides/debugging/images/SSH.tunnel.diagram.svg)
+
+Once you have this web server set up, you many continue to use when ever you want by simply reconnecting to it via the secure tunnel.
```

Comment:
```suggestion
Once you have this web server set up, you may continue to use it whenever you want by simply reconnecting to it via the secure tunnel.
```

Reply:
@mrjones-plip here's the the [editable version](https://docs.google.com/presentation/d/1j4jPsi-gHbiaLBfgYOyru1g_YV98PkBrx2zs7bwhoEQ/edit#slide=id.p) of the diagram Marc linked to in case it's useful; I can also take a stab at it if you prefer!